source/sync/app-builder/device-sync-permissions-guide.txt

```diff
@@ -6,6 +6,14 @@ Device Sync Permissions Guide
 
 .. default-domain:: mongodb
 
+.. meta:: 
+  :description: Learn how to configure the different permissions available to your users using Device Sync.
+  :keywords: code example
+
+.. facet::
```

Comment:
```suggestion
  :keywords: code example

.. facet::
   :name: programming_language
   :values: javascript/typescript
   
.. facet::
```
since we note that there are code examples on the page (with the `code examples` keyword), let's also specify the language used with the [`programming_language` facet](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions?_ga=2.65137646.1361866927.1720454905-28793968.1708561235#Taxonomytagginginstructions-RSTfacet:~:text=Taxonomy%20Genres.-,programming_language,-Driver%20%26%20SDK%20directories).